flow360/component/simulation/simulation_params.py

```diff
@@ -316,6 +317,11 @@ def check_output_fields(params):
         """Only allow lowMachPreconditioner output field when the lowMachPreconditioner is enabled in the NS solver"""
         return _check_output_fields(params)
 
+    @pd.model_validator(mode="after")
+    def check_and_add_is_mrf_flag_in_volumezones(params):
```

Comment:
```suggestion
    def check_and_add_noninertial_reference_frame_model_flag_in_volumezones(params):
```

Reply:
Also change the docstring please to be consistent.

Reply:
Fixed. Thanks.